tools/srs-utils/README.md

```diff
@@ -1,41 +1,102 @@
-# srs-verification
+# SRS Utilities
 
-This project is written for EigenDA SRS related works. It includes
-1. Extract G1 and G2 points used by EigenDA from ptau challenge file, created from the perpetual power of tau MPC instance run by EF
-2. Verify the extracted points are indeed correct based on approaches used by EF KZG ceremony
-3. Provide utility tools for miscellaneous tasks
+This project provides tools for working with EigenDA's Structured Reference String (SRS). It includes tools to:
 
-## To download the ptau file
+1. Download pre-processed SRS files directly from the EigenDA S3 bucket
+2. Extract G1 and G2 points used by EigenDA from the ptau challenge file, created from the Perpetual Powers of Tau MPC ceremony run by the Ethereum Foundation
+3. Verify that the extracted points are correct based on approaches used in the Ethereum Foundation's KZG ceremony
 
-`wget https://pse-trusted-setup-ppot.s3.eu-central-1.amazonaws.com/challenge_0085`
+## Structured Reference String (SRS) Files
 
-See more information from 
-1. https://docs.axiom.xyz/docs/transparency-and-security/kzg-trusted-setup
-2. https://github.com/privacy-scaling-explorations/perpetualpowersoftau/tree/master 
+The SRS files are required for KZG commitments and proofs in EigenDA.
 
-The challenge file has 103079215232 Bytes.
+### File Information
+
+| File Name          | Size   | Number of Points | Point Size | SHA256 Hash                                                      |
+|--------------------|--------|------------------|------------|------------------------------------------------------------------|
+| g1.point           | 16 MB  | 524,288          | 32 bytes   | 8f18b9c04ed4bddcdb73001fb693703197328cecabdfa9025f647410b0c50d7f |
+| g2.point           | 32 MB  | 524,288          | 64 bytes   | a6942684aa751b4ec7873e2edb4660ac5c4516adb3b310441802cc0d489f645a |
+| g2.trailing.point  | 32 MB  | 524,288          | 64 bytes   | 78fad17d74d28cecdb7f826fdd72dee08bdbe1e8ad66f2b24fcf2fc140176788 |
+
+These files represent only a portion of the total SRS data that exists for EigenDA. They are sufficiently large
+to support the largest permitted blob size of 16MB. This maximum blob size may increase in the future,
+at which point larger SRS files will be needed.
+
+Note that the G2 point files (`g2.point` and `g2.trailing.point`) are twice the size of the G1 point file because G2 
+points require twice as many bytes to represent as G1 points in the BN254 curve. Each G1 point requires 32 bytes 
+of storage, while each G2 point requires 64 bytes.
 
 ## How to use
 
 `go run main.go help`
 
-###  How to parse G1, G2 points from the challenge file.
+### Downloading SRS Files
+
+The simplest way to get the required SRS files is to download the pre-processed files directly from the EigenDA
+S3 bucket:
+
+```bash
+go run main.go download --blob-size-bytes 16777216
```

Comment:
I prefer `go install github.com/Layr-Labs/eigenda/tools/srs-utils@latest` and then just run `srs-utils <ARGS>`.
I don't like when instructions force me to clone the repo to run a little CLI binary.